doc/decisions/decision_process.md

```diff
@@ -0,0 +1,137 @@
+- All decisions must go through at least two review rounds, with a merge in between.
+- At least two people need to be in favor of the decision in both rounds.
+- [Documentation guidelines](/doc/contrib/documentation.md) apply.
+- During the decision process, the PRs constantly get updated:
+  - Make changes as new commits to the pull request.
+  - Questions in the PRs are answered by updating the PR and answering with the commit id.
+  - As generally recommended in Elektra, do not squash after they are visible on the pull request.
+  - Rebase only if the decision was already accepted and has a merge conflict.
+- For reviewers:
+  - Prefer to directly give suggestions how to change sentences.
+  - General points should be asked in the root of "Conversation" and not at random sentences in the review.
+
+## Assumptions
+
+- People want to be informed about or even participate in what Elektra looks like in the future.
+- We will always be able to reach an consensus.
+  We don't need a vote or a benevolent dictatorship.
+- Unlike the Rust Decision process, decisions in Elektra do not have a disadvantage if they were flawed in early stages.
+  Only the end results counts.
+
+## Considered Alternatives
+
+- Issues like https://issues.libelektra.org/4521
+- PEPs: https://peps.python.org
+- RFCs: https://www.ietf.org/standards/rfcs/
+- Change requests: https://en.wikipedia.org/wiki/Change_request
+
+## Decision
+
+Decisions need to:
+
+- be implementable within the next major release
+- be according to [Elektra's goals](/doc/GOALS.md)
+- first being decided upon using the decision process described here
+
+We base our decision process and template on:
+
+- [''using patterns to capture architectural decisions''](https://dl.acm.org/doi/10.1109/MS.2007.124),
+- [arc42 decisions](http://docs.arc42.org/section-9/),
+- [ADR](https://adr.github.io/), and
+- [RFCs in rust-lang](https://github.com/rust-lang/rfcs).
+
+Following subsections describe all steps a decision might run through.
+Only two of them are mandatory.
+
+Explanations of the template are in [explanations.md](explanations.md).
+
+### Drafts
+
+The first step is to create a PR with:
+
+- **one** decision copied from [template.md](template.md), where at least the "Problem" and "Considered Alternatives" are filled out.
+- a link from [README.md](README.md) from the "Drafts" section to this decision.
+
+### In Discussion
+
+This step is mandatory.
+
+Here you must ensure:
+
+- consistency with other decisions
+- links from/to related decisions are created
+- problem, constraint and assumptions are fully described and sound
+- there are several considered alternatives, each with rationale and implication
+- decision, rationale and implications is **not** yet filled out if there are people arguing for different options (to keep the discussion unbiased)
+
+Here "the decision" should not have one decision but several well-described solutions.
```

Comment:
> several well-described solutions

Where do these come from? Sounds like author has to propose multiple solution. Which would be a waste of time, if everybody already agrees on one solution.

Reply:
Not quite what I meant. The issue is mostly the "well-described" part. It sounds like I have to completely think through at least two solutions, even if it is very obvious that one choice is better than the other.

Maybe something like

```suggestion
Here "the decision" should not have one decision but should describe multiple possible solutions.
The possible solutions have to be described well enough to show why the decision was made.
```

This still means we need multiple solutions, but if there is one very obvious best solution the other don't need to be described in detail.